README.md

```diff
@@ -1 +1,70 @@
-# Math-magicians-website
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly edit the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) template which include `Authors`, `Contributing`,  `Acknowledgments`.